notebooks/224-distilbert-Sequence-Classification/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Sequence Classification With OpenVINO API 2.0
+This notebook performs sequence classification using OpenVINO API 2.0. You can alternatively replace the model with the other bert-based models for sequence classification.
+
+Output from a user input would look like this.
+
+![image](https://user-images.githubusercontent.com/95271966/203713415-669a755d-3243-4e66-b19b-cd17774a1a64.png)
+
+
+Output from a file would look something like this.
+![image](https://user-images.githubusercontent.com/95271966/203713154-b78b383e-ec42-4b3b-a142-47b00640bdea.png)
+
+
+## Notebook Contents
+We'll be using distilbert-base-uncased-finetuned-sst-2-english transformers based model from Huggingface. Details of the model can be found [here].(https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)

```

Comment:
I made suggestion to put it to description as it is general info about notebook. I think here should be answer what this notebook do?
Like:
 this notebook demostrates how to convert model to ONNX format, then OpenVINO Intermediate Representation, then run model inference on user specified input text and from file for performing sentiment analysis.